MapboxCoreNavigation.podspec

```diff
@@ -40,7 +40,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 48.0"
+  s.dependency "MapboxNavigationNative", "~> 49.0.1"
```

Comment:
Do we need to be this specific, or can we require just v49._x_ and rely on the dependency manager to get v49._x_ unless the developer specifically asks for v49.0.0 (which would be unlikely in this case)?